src/main/kotlin/com/yapp/lettie/api/file/service/FileService.kt

```
@@ -0,0 +1,99 @@
+
+            val presignRequest =
+                GetObjectPresignRequest
+                    .builder()
+                    .signatureDuration(Duration.ofMinutes(EXPIRY_IN_MINUTES.toLong()))
+                    .getObjectRequest(getObjectRequest)
+                    .build()
+
+            val presigned: PresignedGetObjectRequest = s3Presigner.presignGetObject(presignRequest)
+
+            return PresignedUrlDto(presigned.url().toString(), file.objectKey, expiresAt)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)